ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -167,30 +170,12 @@ public void handleGet(RestRequest restRequest, RestResponseChannel restResponseC
 
   @Override
   public void handlePost(RestRequest restRequest, RestResponseChannel restResponseChannel) {
-    long processingStartTime = System.currentTimeMillis();
-    long preProcessingTime = 0;
     handlePrechecks(restRequest, restResponseChannel);
```

Comment:
Is it still necessary to do this precheck, since the method itself is not supported, maybe directly jump to generate error msg? Another thought is that if this part throws exception, it may hide the information that this method is not supported, otherwise the caller would supply the required parameter and try again.